### DISCUSSION TITLE
Adding builtin Tiptap extensions

### DISCUSSION THREAD

**Discussion by IlaySkutelsky** [awcodes/filament-tiptap-editor#527] [Q&A]
Hi! Thank you for the work on this awesome library.
I want to include the [builtin Tiptap Mention extension](https://tiptap.dev/docs/editor/extensions/nodes/mention) through the config file.
Even before touching the custom JS and CSS files, I just need to understand how to register it in the config file.
I tried:
`
    'extensions' => [
        new \Tiptap\Nodes\Mention(),
    ]
...
    'extensions' => [
       'Mention',
    ],
` 
But both threw errors. Did anyone managed to do this? Or can anyone point me to the direction I should take here?

**Comment by awcodes** [comment]
The extensions path in the config is expecting a Vite path for the compiled js file.
